Mogakco/Sources/Data/DataSources/Remote/ChatDataSource.swift

```diff
@@ -80,10 +83,35 @@ final class ChatDataSource: ChatDataSourceProtocol {
             Constant.chatRoom
                 .document(chatRoomID)
                 .collection("chat")
-                .addDocument(data: chat.toDictionary()) { _ in
+                .document(chat.id)
+                .setData(chat.toDictionary()) { _ in
                     emitter.onNext(())
                 }
             return Disposables.create()
         }
     }
+    
+    func read(chat: Chat, userID: String) -> Observable<Void> {
+        return Observable.create { emitter in
+            if !chat.readUserIDs.contains(userID) {
+                var chat2 = chat
+                
```

Comment:
```suggestion
            if !chat.readUserIDs.contains(userID) {
                var readChat = chat
                
```

의미가 들어있지 않은 넘버링이 들어간 변수명은 가독성을 해칠 수 있을 것 같습니당!

Reply:
테스트용으로 임의로 적어놨던 변수였는데 잘되길래 깜빡하고 내버려둔 것 같네욬ㅋㅋ 고치겠습니다